core/block_render_svg.js

```diff
@@ -687,8 +651,8 @@ Blockly.BlockSvg.prototype.renderDrawTop_ = function(pathObject, rightEdge) {
     steps.push(Blockly.BlockSvg.NOTCH_PATH_LEFT);
     highlightSteps.push(Blockly.BlockSvg.NOTCH_PATH_LEFT_HIGHLIGHT);
 
-    var connectionX = (this.RTL ?
-        -Blockly.BlockSvg.NOTCH_WIDTH : Blockly.BlockSvg.NOTCH_WIDTH);
+    var notchOffsetStart = 15;
```

Comment:
Why was the constant replaced with a number? It also appears in two other spots below, so worth pulling back out into a constant.

Reply:
Done, here and for the other constant.

This change moves the connection's position from the right of the notch to the left of the notch.  The notch width constant used above is *not* the width of the notch.  It's the distance from the left of the block to the right of the notch.

The other change moves the connection position from 5 pixels above the puzzle tab to the top of the puzzle tab.

These changes are safe as long as I always pair changes to matching types of connections.